qemu/tests/balloon_stress.py

```diff
@@ -34,27 +77,22 @@ def run(test, params, env):
 
     driver_name = params["driver_name"]
     utils_test.qemu.setup_win_driver_verifier(driver_name, vm, timeout)
-    balloon_test = BallooningTestWin(test, params, env)
-
-    video_play = utils_misc.InterruptedThread(
-        utils_test.run_virt_sub_test, (test, params, env),
-        {"sub_type": params.get("sub_test")})
-    video_play.start()
-    error_context.context("Run video background", logging.info)
-    check_playing_cmd = params["check_playing_cmd"]
-    running = utils_misc.wait_for(
-        lambda: utils_misc.get_guest_cmd_status_output(
-            vm, check_playing_cmd)[0] == 0, first=60, timeout=600)
-    if not running:
-        raise exceptions.TestError("Video is not playing")
-
+    run_video(vm)
```

Comment:
If we doing a memory balloon immediately after playing video, QMP event BALLOON_CHANGE may not be received. It is better to sleep a few seconds.